src/main/webapp/mydata_templates/user_list.html

```diff
@@ -0,0 +1,63 @@
+String getCitation(){
+    > Backing bean
+        > get Dataset
+         to JSON ->     Lots of RestAssuredTEsts
```

Comment:
file removed, this was from the initial version using JSON (piece under pseudo-code was working nunjucks/jinja-like template)